src/fsharp/service/FSharpCheckerResults.fsi

```diff
@@ -234,7 +216,7 @@ type public FSharpCheckFileResults =
     member GetSymbolUseAtLocation  : line:int * colAtEndOfNames:int * lineText:string * names:string list * ?userOpName: string -> Async<FSharpSymbolUse option>
 
     /// <summary>Get any extra colorization info that is available after the typecheck</summary>
-    member GetSemanticClassification : range option -> (range * SemanticClassificationType)[]
+    member GetSemanticClassification : range option -> struct (range * SemanticClassificationType)[]
```

Comment:
Just a note that this will technically be a breaking change, so a corresponding FCS update will need to be a major version bump.

Reply:
we major bump basically every revision (just due to the nature of the passage of time/AST API changes) so that's not a blocker by any means.